Source/Core/DiscIO/VolumeVerifier.cpp

```diff
@@ -313,7 +313,11 @@ std::vector<RedumpVerifier::PotentialMatch> RedumpVerifier::ScanDatfile(const st
     // so show a panic alert rather than just using ERROR_LOG
 
     // i18n: "Serial" refers to serial numbers, e.g. RVL-RSBE-USA
-    PanicAlertT("Serial and/or version data is missing from %s", GetPathForSystem(system).c_str());
+    PanicAlertT("Serial and/or version data is missing from %s\n"
+                "Please append \"%s\" (without the quotes) to the datfile URL when downloading\n"
+                "(For instance, %s)",
```

Comment:
Nitpicking, but maybe just `Example: %s` (without the parentheses) would be sufficient, since it's on its own line?